Makefile

```diff
@@ -192,18 +202,13 @@ $(VERSION_FILE):
 	cd $(IMAGE_BUILD_FOLDER) &&	make image PROFILE="$$(cat $(IMAGE_BUILD_FOLDER)/profile.build.tmp )" PACKAGES="$(GENERAL_PACKAGES)" FILES=$(FILES_FOLDER)
 ifneq ($(INSTALL_PREFIX),)
 	mkdir -p $(INSTALL_PREFIX)
-	cp $(IMAGE_BUILD_FOLDER)/bin/targets/$(TARGET)/$(TARGET_TYPE)/$@ $(INSTALL_PREFIX)/$(OPENWRT_COMP)$@
+	cp $(IMAGE_BUILD_FOLDER)/bin/targets/$(TARGET)/$(TARGET_TYPE)/*squashfs-factory.bin $(INSTALL_PREFIX)/$(OPENWRT_COMP)$@
```

Comment:
Q/RFC:
I am not quite sure if I want to copy over always all generated images. The corresponding make rule will copy over the .bin file which it is currently working on.